source/renderer/app/containers/wallet/dialogs/WalletSendConfirmationDialogContainer.js

```diff
@@ -16,13 +17,14 @@ type Props = {
   receiver: string,
   assets: Array<AssetToken>,
   assetsAmounts: Array<string>,
-  totalAmount: ?string,
+  totalAmount: BigNumber,
```

Comment:
@danielmain we need to be careful with this change. It could impact the HW interaction. We need to check if this value is passed to HW or cardano-wallet API.
cc @tomislavhoracek @dmitrii-gaico

Reply:
@nikolaglumac this is only a type change in flow. `totalAmount` is only used in this component (WalletSendAssetsConfirmationDialog) , and his parent WalletSendConfirmationDialogContainer **nowhere else**.

Reply:
Alright then! Perfect 👌